workflows/steps/tasks/addfeaturetask.js

```diff
@@ -124,6 +142,7 @@ proto.stop = function() {
   this.drawInteraction = null;
   this.drawingFeature = null;
   document.removeEventListener('keydown', this._delKeyRemoveLastPoint);
+  this._stopPromise.resolve(true);
```

Comment:
why not?

```suggestion
  d.resolve(true);
```

Reply:
@Raruto need to wait stop call method to set `this._stopPromise.resolve(true)`;. this._stopPromise is passed as parameter promise to
```
 this.setAndUnsetSelectedFeaturesStyle({
        promise: this._stopPromise
      });
```
to setAndUnsetSelectedFeaturesStyle method. We need to wait stop promise resolve because task can stop after run resolve promise, or if ESC key is pressed